### PR TITLE
add introduction

### PR DESCRIPTION
**Pull request by jacquesqiao**

### CODE REVIEW

File: docs/introduction/introduction.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+上述只是Visual DL诸多功能中的一项，Visual DL有以下几方面的优点。
+
+### 功能多
+1. Scalar. 支持打点数据Scalar展示，如上图所示：
+	- 可以将训练过程中的loss，准确率等信息以折现的形式体现出来，方便看出整体趋势。
+	- 可以在同一个图片中画出多条曲线，方便对比分析。
+2. Image. 支持图片展示。
+	- 输入图片展示，方便查看数据样本的质量
+	- 支持卷积层的结果展示，方便查看卷积之后的效果。
+<p align="center">
```
Comment (Superjomn) [inline docs/introduction/introduction.md:29]: add a gan RGB image will be more attractive.

File: docs/introduction/introduction.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+<img src="histogram.png" />
+</p>
+
+4. Graph.  方便查看深度神经网络的模型结构。
+	- 便于排查网络配置的错误
+	- 帮助理解网络结构
+
+<p align="center">
+<img src="graph.png" height="250" width="260"/>
+</p>
```
Comment (Superjomn) [inline docs/introduction/introduction.md]: add a link for the original graph image, add to git if the image file is not large.

File: docs/introduction/introduction.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+loss:[2.6916795] acc:[0.09375] pass_acc:[0.08984375]
+loss:[2.7038598] acc:[0.1171875] pass_acc:[0.09895834]
+loss:[2.6405232] acc:[0.125] pass_acc:[0.10546875]
+```
+
+这种方式可以短期看一下训练状态，但是很难从全局把握训练过程的一些变化趋势，提取的信息有限。
+
+Visual DL 让你方便的将训练过程可视化起来，形成类似下面的图片：
+<p align="center">
+<img src="scalar.png"/>
```
Comment (Superjomn) [inline docs/introduction/introduction.md:17]: scalar need 0.8 smooth, or it looks too steep.

Comment (jacquesqiao) [reply docs/introduction/introduction.md:17]: done

File: docs/introduction/introduction.md (hunk 1 above)
Comment (jacquesqiao) [reply docs/introduction/introduction.md:29]: done

**Review by daming-lu — APPROVED** [review]

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**